README.md

```diff
@@ -202,6 +203,7 @@ cisco_interface_ospf {"Ethernet1/2 Sample":
 * [`cisco_vlan`](#type-cisco_vlan)
 * [`cisco_vrf`](#type-cisco_vrf)
 * [`cisco_vtp`](#type-cisco_vtp)
+* [`network_interface`](#type-network_interface)
```

Comment:
IMO, this should be under Interface Types and we are adding the netdev_stdlib tag.

 * \[`network_interface (netdev_stdlib)`\](#type-network_interface)